src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/exception/BadRequestException.kt

```diff
@@ -0,0 +1,7 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.exception
```

Comment:
this isn't nessecary. spring has one already https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/server/ServerWebInputException.html

Reply:
updated